server/src/server.ts

```diff
@@ -588,6 +588,33 @@ function format(msg: p.RequestMessage): Array<p.Message> {
   }
 }
 
+const updateDiagnosticSyntax = (fileUri: string, fileContent: string) => {
+  const filePath = fileURLToPath(fileUri);
+  const tmpname = utils.createFileInTempDir();
```

Comment:
Other commands:
```rescript
  let extension = path.extname(params.textDocument.uri);
  let code = getOpenedFileContent(params.textDocument.uri);
  let tmpname = utils.createFileInTempDir(extension);
```
that's how you get the extension right, to be used by the command.